### PR TITLE
Allow serde 0.7

### PR DESCRIPTION
**Pull request by nox**
We now use bincode from crates.io.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-version = "0.2.0"
+version = "0.2.1"
 
 [lib]
 name = "ipc_channel"
 path = "lib.rs"
 
-[dependencies.serde]
-version = "0.6"
-opt_level = 2
```
Comment (antrik) [inline Cargo.toml]: Why did you remove this?

Comment (nox) [inline Cargo.toml]: Because it doesn't do anything according to Cargo.
